Packs/Microsoft365Defender/Classifiers/classifier-Microsoft_365_Defender_-_Incoming_Mapper.json

```diff
@@ -0,0 +1,344 @@
+{
+    "description": "Classifies Microsoft 365 Defender's events",
+    "feed": false,
+    "fromVersion": "6.0.0",
```

Comment:
Like layouts, there are also classifiers for <5.9.9 we need to create one here and generate mapping for it - you can look at the XDR's pack for an example.

Reply:
Done